src/app/core/feedback/feedback.controller.js

```diff
@@ -82,29 +82,24 @@ module.exports.adminGetFeedbackCSV = async function (req, res) {
 		});
 
 		const query = result.config.q ? JSON.parse(result.config.q) : null;
-		const sortArr = [
-			{ property: result.config.sort, direction: result.config.dir }
-		];
-
-		const feedbackResult = await Feedback.textSearch(
-			query,
-			null,
-			null,
-			null,
-			sortArr,
-			true,
-			{
+		const search = result.config.s;
+		const sort = utilService.getSortObj(result.config);
+
+		const feedbackCursor = Feedback.find(query)
+			.textSearch(search)
+			.sort(sort)
+			.populate({
 				path: 'creator',
 				select: ['username', 'organization', 'name', 'email']
-			}
-		);
+			})
+			.cursor();
 
 		exportConfigController.exportCSV(
 			req,
 			res,
 			exportFileName,
 			columns,
-			feedbackResult.results
+			feedbackCursor
 		);
```

Comment:
This shows another advantage.  When we can still use the `textSearch` functionality, without having to pass in a bunch of nulls to get all of the data vs. a single page.  It also allows us to use mongoose cursors to avoid having to load all of the data into memory at once.